packages/esm-patient-registration-app/src/patient-registration/patient-registration.scss

```diff
@@ -91,6 +103,18 @@
   .infoGrid {
     max-width: unset;
   }
+
+  .space05 {
+    padding: 13px 0;
+  }
+
+  .linkContainer {
+    div:last-child {
+      padding-bottom: 2.3rem;
```

Comment:
Weird

Reply:
> Weird

this is responsible for the lower grey line after the `links` and the `Cancel` `Register` buttons. Different spacing on tablet and desktop
![image](https://github.com/openmrs/openmrs-esm-patient-management/assets/104269786/3783685c-0904-4e15-b5cb-6ce03a1130c7)
![image](https://github.com/openmrs/openmrs-esm-patient-management/assets/104269786/2214a9a9-b3c3-43ca-8e16-bf1cd50e77a5)